hazelcast-client/src/main/resources/hazelcast-client-config-3.5.xsd

```diff
@@ -23,7 +23,7 @@
 
     <xs:element name="hazelcast-client">
         <xs:complexType>
-            <xs:sequence>
+            <xs:choice minOccurs="1" maxOccurs="unbounded">
```

Comment:
can you please expalin in one sentence what is the diff between **<xs:choice minOccurs="1" maxOccurs="unbounded">** and **<xs:all>**

I forget it all the time.